TreasureHunt/Assets/2.Scripts/Hunter/FieldView/TreasureSetupController.cs

```diff
@@ -4,64 +4,68 @@
 using System.Globalization;
 using SimpleJSON;
 
-public class TreasureSetupController : MonoBehaviour {
-
+public class TreasureSetupController : MonoBehaviour 
+{
 	public static TreasureSetupController instance = null;
+	public static string userGameTreasureData = "";
 
 	void Awake()
 	{
-		if (instance == null)
-
+		if (instance == null) 
+		{
 			instance = this;
-
-		else if (instance != this)
-
-			Destroy(gameObject);
-
-		DontDestroyOnLoad (gameObject);
-
-	}
-
-	void Start(){
-		if (!gameObject.GetComponent<NetworkManager> ().enabled) {
-			GetGameTreasure ("gg");		
+		} 
+		else if (instance != this) 
+		{
+			Destroy (gameObject);
 		}
 	}
 
-	public void GetGameTreasure(string userName){
+	void Start()
+	{
+		// Todo: get user id and pass it as an argument, instead of gg
+		GetGameTreasure ("gg");
+	}
 
-		string str = "{\"flag\":3, \"usn\":\"" + userName + "\"}";
-		if (!gameObject.GetComponent<NetworkManager> ().enabled) {
+	public void GetGameTreasure(string userName)
+	{
+		if (!gameObject.GetComponent<NetworkManager> ().enabled) 
+		{
 			TextAsset jsonData = Resources.Load<TextAsset> ("TestForTreasureSetup");
 			var strJsonData = jsonData.text;
 			Debug.Log (strJsonData);
 			ForEachGame (strJsonData);
-		} else {
+		}
+		else 
+		{
+			string str = "{\"flag\":3, \"usn\":\"" + userName + "\"}";
 			NetworkManager.instance.SendData (str);
 		}
-
 	}
 
 	//infos included in json:
 	//	flag,
-	//	game_id, game_name, treasure_count, maker_id, status,
+	//	game_id, game_name, treasure_count, maker_id, status, participant
 	//	treasure_id, treasure_name, description, game_id, location, point, catchgame_cat, target_img_url
 	public GameObject game;
 	public GameObject treasure;
 	public static List<string> m_Data = new List<string>();
```

Comment:
Oh, I'm specifically talking about the List<string> data member. The data type is written as `m_Data`. Why not just `data`?